client/post-editor/media-modal/detail/_style.scss

```diff
@@ -97,6 +97,16 @@
 	}
 }
 
+.editor-media-modal-detail__edit {
+	width: 100%;
+	margin-bottom: 16px;
+	font-weight: bold;
+}
+
+.editor-media-modal-detail__edit-text {
```

Comment:
This class should be unnecessary because `Button` should already accommodate gridicons if written like:
`<Button><Gridicon /> Text</Button>`